tests/integration/common-operator-integ-suite.sh

```diff
@@ -172,13 +177,33 @@ main_test() {
     ${COMMAND} get ns "${CONTROL_PLANE_NS}" >/dev/null 2>&1 || ${COMMAND} create namespace "${CONTROL_PLANE_NS}"
     sed -e "s/version:.*/version: ${ver}/g" "${ISTIO_MANIFEST}" | ${COMMAND} apply -f - -n "${CONTROL_PLANE_NS}"
 
+    echo "Wait for Istio to be Reconciled"
+    ${COMMAND} wait istio/istio-sample -n "${CONTROL_PLANE_NS}" --for condition=Reconciled=True --timeout=${TIMEOUT}
+
+    echo "Wait for Istio to be Ready"
+    ${COMMAND} wait istio/istio-sample -n "${CONTROL_PLANE_NS}" --for condition=Ready=True --timeout=${TIMEOUT}
+
+    echo "Give the operator some time to settle down"
+    sleep 30
+
+    echo "Check that the operator has stopped reconciling the resource"
```

Comment:
nit: append sth like "Waiting 30s"

just so that people running this test manually knows what to expect

Reply:
The tests waits in a few other places as well, and it doesn't say how long...   but I've made the change as suggested